### PR TITLE
Basic script for configuring terraform through environment vars (prelude to CI)

### PR DESCRIPTION
**Pull request by grafnu**

### CODE REVIEW

File: cloud/gcp/terraform.tfvars.template [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 gcp_auth_file = "./auth/credentials.json"
 
 # Storage Buckets
-tf-state-bucket-name   = "udmi-terraform-state-bucket"
+tf-state-bucket-name   = "@GCP_PROJECT_NAME@-terraform"
 tf-state-storage-class = "STANDARD"
 
 # Access
-udmi_access_group = "group:group@example.com" # CHANGE ME
+udmi_access_group = "group:@UDMI_SITE_GROUP@"
```
Comment (pisuke) [inline cloud/gcp/terraform.tfvars.template:12]: This group account is to give admin access to the resources, so it should have a different variable than the one in the udmi-sites.tf file

File: cloud/gcp/udmi-sites.tf.template [hunk 2]
```diff
@@ -1,8 +1,8 @@
 module "ZZ-TRI-FECTA" {
   source      = "./modules/terraform-google-udmi-site"
   gcp_project = var.gcp_project_name
-  site_name   = "ZZ-TRI-FECTA"
-  site_region = "europe-west1"
-  site_group  = "group:group@example.com"
+  site_name   = "@UDMI_SITE_NAME@"
+  site_region = "@UDMI_SITE_REGION@"
+  site_group  = "group:@UDMI_SITE_GROUP@"
```
Comment (pisuke) [inline cloud/gcp/udmi-sites.tf.template:6]: This group account is to give only viewer/provisioning access to IoT Core registries and subscriber access to the pub/sub topic resources, so it should have a different variable than the one in the variables template file

Comment (grafnu) [reply cloud/gcp/udmi-sites.tf.template:6]: done

File: cloud/gcp/terraform.tfvars.template (hunk 1 above)
Comment (grafnu) [reply cloud/gcp/terraform.tfvars.template]: done

**Review by pisuke — APPROVED** [review]
LGTM, thanks!

**Pull request merged**